### DISCUSSION TITLE
Is there a way to see and change the view count?

### DISCUSSION THREAD

**Discussion by fdw** [jellyfin/jellyfin#9342] [Q&A]
The Trakt plugin sometimes adds multiple viewings of the same stream at the same time. [According to its bug tracker](https://github.com/jellyfin/jellyfin-plugin-trakt/issues/162), it simply follows Jellyfin's playcount. So now I'm wondering where I can find and change the playcounts. It's fine if it's in a DB somewhere, but I haven't found anything on my own.

**Comment by mihawk90** [comment]
It's not visible in the UI from what I know, but the playcount is kept in the database.
You can open the database with anything that can read SQLite (I used `sqlitebrowser`) and the playcount is kept in the `UserDatas` table. However, the playcount is complete bogus:

key | userId | rating | played | playCount | isFavorite | playbackPositionTicks | lastPlayedDate | AudioStreamIndex | SubtitleStreamIndex
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
ilan Bluestone-Impulse-0002-0016Love Is a Drug (extended mix) | 1 |   | 1 | 1763 | 0 | 0 | 2023-02-27 16:25:11.9368729Z |   |  
b174c1f3-f262-319e-0345-3dc03a208f50 | 1 |   | 1 | 1763 | 0 | 0 | 2023-02-27 16:25:11.9368729Z |   |  

Needless to say there is no way I played this track 1763 times in the 3 weeks that I owned it.

On that same note the entire album has wildly differing playcounts between 14 and 30 (ignoring the one above), even though I exclusively listen to entire albums and I listened to this maybe 3? 4? times.

 And this isn't the only track with extremely high playcounts. What I do notice though is that it always seems to be the last track on an album that has these extreme numbers. The other tracks aren't correct either but the last one seems to be the one that is way off.
